webpack.config.js

```diff
@@ -9,15 +9,17 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 require('dotenv').config()
 const uuidv4 = require('uuid/v4')
 
+const ProcessUtils = require('./common/processUtils')
+
 const mode = {
   development: 'development',
   production: 'production'
 }
 
-const prodBuild = process.env.NODE_ENV === mode.production
-const buildReport = process.env.BUILD_REPORT === 'true'
+const prodBuild = ProcessUtils.ENV.nodeEnv === mode.production
```

Comment:
can we add ProcessUtils.envProduction ? (internal mode object should be removed)